frontend/src/components/input/file-upload/FileUpload.vue

```diff
@@ -424,12 +424,25 @@ const runModelHandler = async () => {
 
     const response = await projectionHcsvPost(getFormData(), false)
 
+    console.log('Full response:', response)
```

Comment:
Do we need to console logouts in this file? Or are you using them for some debugging?